src/output-json.c

```diff
@@ -150,6 +150,9 @@ static enum JsonOutput json_out = ALERT_FILE;
 
 static enum JsonFormat format = COMPACT;
 
+/* Prepend @cee: cookie to eve JSON syslog output for CEE/Lumberjack compatibility */
+static int syslog_cee_format = FALSE; /* Default OFF */
```

Comment:
On 05/20/2015 03:03 PM, Zach Rasmor wrote:
> Would you like me to open a new pull request for this feature when
> @regit <https://github.com/regit> finishes his refactor?
> 

Yes please. This has happened now, so the floor is yours!